### DISCUSSION TITLE
科技玩家 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#51] [模板讨论]
科技玩家 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by zadishu** [comment]
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 520: Unknown,
Request URL: https://www.kejiwanjia.net/wp-json/jwt-auth/v1/token

**Reply by wjf0214** [reply]
HTTP 520: Unknown ，应该是你请求网站失败了。可能当时网站有点问题，后续再看看。如果还不行建议检测下网络设置。

**Reply by 654850695** [reply]
2024-03-15 17:39:20
日志：Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 301: Moved Permanently,
Request URL: https://www.kejiwanjia.net/wp-json/jwt-auth/v1/token
